requirements-dev.txt

```diff
@@ -12,9 +12,9 @@ black==24.10.0
     # via -r requirements-dev.in
 boolean-py==4.0
     # via license-expression
-boto3-stubs[cognito-idp,dynamodb,events,lambda,s3,sns,sqs,ssm]==1.34.158
+boto3-stubs[cognito-idp,dynamodb,events,lambda,s3,sns,sqs,ssm]==1.35.60
     # via -r requirements-dev.in
-botocore-stubs==1.34.158
+botocore-stubs==1.35.60
```

Comment:
We have returned to what I pointed out in the previous pull request: https://github.com/pdyba/lambdalizator/pull/102#discussion_r1842986399

> you need to decide whether to use boto3 in version 1.35.xx or boto3-stubs compatible with 1.34.xxx 😉

Reply:
Maybe, it is a good moment to bump it to 1.35.xx if you want that?